src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/LegacyReleaseController.cs

```diff
@@ -18,16 +20,24 @@ public LegacyReleaseController(ILegacyReleaseService legacyReleaseService)
         {
             _legacyReleaseService = legacyReleaseService;
         }
-        
-        [HttpGet("api/legacy-releases/{id}")]
+
+        [HttpGet("publications/{publicationId}/legacy-releases")]
```

Comment:
Can use the `:guid` request parameter hint here to ensure it's the right type.

Reply:
Same for other new Controller methods

Reply:
Thanks, made this change.